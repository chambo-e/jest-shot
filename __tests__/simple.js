import shot from '../src/';
import SimpleComponent from '../__components__/simple';

shot(SimpleComponent);
