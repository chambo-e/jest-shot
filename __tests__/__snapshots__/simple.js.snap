// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleComponent renders correctly without props 1`] = `
<div>
  Hello
</div>
`;
